### DISCUSSION TITLE
Domain Ownership Transfer Request

### DISCUSSION THREAD

**Discussion by Devenvaruv** [vercel/community#4068] [Help] [answered]
### Summary

Hello Vercel Support Team, I am writing to request assistance with a domain ownership transfer issue that we are currently facing. Here is the situation:

There is a repo named Interest-form and one of my colleague deployed it from their personal vercel account. but now they are no longer associated with us so before leaving we forked their repo. Now when we are trying to deploy the same repo Interest-form to our vercel account it gives us an error because the domain name is being used by their personal vercel account. if they remove their repo from their personal vercel account when will the domain name be available? 

### Steps to Reproduce

This is the error message that we are getting.

Cannot add interest-form.vercel.app since it's already assigned to another project.

**Comment by amyegan** [comment, marked as answer]
Hey @Devenvaruv! Your colleague can transfer the domain to you by following the steps on this page: https://vercel.com/docs/projects/domains/transfer-your-domain

**Reply by Devenvaruv** [reply]
Thank you so much!!!
